src/app/header/header.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss']
+})
+
+export class HeaderComponent {
+  title = 'Angular ToDo App'
```

Comment:
Use i18n for all titles and text values. In component  use keys with translate pipe